assets/sass/components/global/_googlesitekit-help-menu.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * HelpMenu styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin .googlesitekit-help-menu {
+
+	margin-right: $grid-gap-phone / 2;
+	padding: 0;
+	width: 52px;
```

Comment:
Can you elaborate on why this is needed? Seems like an arbitrary fixed value.

Also, are there really any styles which are specific to the help menu? I would all 3 menus should largely share the same styles.

Reply:
Actually, you're right, it's no longer needed 👍🏻